quantum/os_detection.c

```diff
@@ -45,41 +50,88 @@ struct setups_data_t setups_data = {
     .cnt_ff = 0,
 };
 
-os_variant_t detected_os = OS_UNSURE;
+#    ifndef OS_DETECTION_DEBOUNCE
+#        define OS_DETECTION_DEBOUNCE 200
+#    endif
+
+// 2s should always be more than enough
```

Comment:
Half tempted to want to add a "otherwise, you may have other issues)" comment

Reply:
Done ;)